package.json

```diff
@@ -26,7 +26,7 @@
     "@angular/router": "^8.0.0",
     "@angular/upgrade": "^8.0.0",
     "@mapbox/stylelint-processor-arbitrary-tags": "^0.2.0",
-    "@types/angular": "^1.6.54",
+    "@types/angular": "^1.7.9",
```

Comment:
This is not ready for upgrade yet, please revert. @ankita240796 is working on a PR for this.

Reply:
if you have having trouble pushing, please make sure you pull from the latest develop, we toggled the test.

Reply:
Noted. Sorry about that, @kevinlee12! I'll fix that right now.

Reply:
no worries, thanks!